src/test/java/Homework17.java

```diff
@@ -0,0 +1,60 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework17 extends BaseTest {
+
+    @Test
+    public void addSongToPlaylist() {
+      openLoginUrl();
+      login();
+      createPlaylist();
+      clickAllSongs();
+      addFirstSongToPlaylist();
+      Assert.assertTrue(msgPopup());
+    }
+
+public boolean msgPopup(){
+        WebElement msg = driver.findElement(By.cssSelector("div[class='success show']"));
+        return(msg.isDisplayed());
+
+}
+public void createPlaylist() {
+    WebElement playlistbtn = driver.findElement(By.cssSelector("i[title='Create a new playlist']"));
+    playlistbtn.click();
+    WebElement playlistcontextmenu = driver.findElement(By.cssSelector("li[data-testid='playlist-context-menu-create-simple']"));
+    playlistcontextmenu.click();
+    WebElement inputplaylistname = driver.findElement(By.cssSelector("input[name='name']"));
+    inputplaylistname.sendKeys("Test");
+    inputplaylistname.sendKeys(Keys.ENTER);
+
+}
+public void searchinput(){
+        //input[type='search']
+    WebElement searchfield = driver.findElement(By.cssSelector("input[type='search']"));
+    searchfield.click();
+    searchfield.sendKeys("dark");
+    searchfield.sendKeys(Keys.ENTER);
+    WebElement allsongsbnt = driver.findElement(By.cssSelector("button[data-test='view-all-songs-btn']"));
+    allsongsbnt.click();
+
+}
+    public void clickAllSongs() {
+        WebElement allsongs = driver.findElement(By.cssSelector("a[href='#!/songs']"));
+        allsongs.click();
+    }
+    public void addFirstSongToPlaylist(){
+        WebElement song = driver.findElement(By.xpath("//tr/td[4]"));
+        song.click();
+        WebElement addbtn= driver.findElement(By.cssSelector(".btn-add-to"));
+        addbtn.click();
+        WebElement playlistbtn = driver.findElement(By.cssSelector("li[class='playlist']"));
+        playlistbtn.click();
+    }
+
+
+
+
```

Comment:
**Please delete the empty line!** An Empty Line is a "Code Smell"
The short answer is, _They’re more an annoyance than a problem.They won’t actually break anything—which is why your code continues to run—but they make it awkward to read and change your code, especially if you’re dealing with multiple line breaks per line of real code. They also add just a tiny bit of size to your files, which is nice to reduce._